tasks/compile-example.js

```diff
@@ -57,13 +57,16 @@ module.exports = function(templateRoot, template) {
       const document = DocumentParser.parse(contents);
       const stream = this;
       const title = FileName.toString(file);
+      const next = FileName.nextFile(file);
 
       const args = {
         head: document.head,
         title: title,
         subHeading: title,
         exampleStyles: document.styles,
-        sections: document.sections
+        sections: document.sections,
+        next: next,
+        nextTitle: gutil.replaceExtension(next, '')
```

Comment:
otherwise samples with special chars will break, you can test it with the following sample:

```
gulp create -n "What's up?"
```